src/EventStore.Core.Tests/Services/Transport/Tcp/TcpClientDispatcherTests.cs

```diff
@@ -27,17 +27,21 @@ public class TcpClientDispatcherTests {
 
 		private ClientTcpDispatcher _dispatcher;
 		private TcpConnectionManager _connection;
+		private ClusterVNodeOptions.DefaultUserOptions _defaultUserOptions;
 
 		[OneTimeSetUp]
 		public void Setup() {
 			_dispatcher = new ClientTcpDispatcher(2000);
-
+			_defaultUserOptions = new ClusterVNodeOptions.DefaultUserOptions() {
```

Comment:
Perhaps it would be worthwhile adding the default user options to a common location and reusing them from there? For example, you could add them to DefaultData: https://github.com/EventStore/EventStore/blob/master/src/EventStore.Core.Tests/DefaultData.cs